Assignment4/GNajeral-180275/task07.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""Task07.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1fzywZ4EwNykCSoc5kjbwf7_KAb8NKSC-
+
+**Task 07: Querying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
with this line, this would not execute in python. Please comment it out

Reply:
Fixed